src/main/java/view/InputView.java

```diff
@@ -0,0 +1,180 @@
+package view;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import model.StudentAttendanceRecord;
+import model.TodayDate;
+
+public class InputView {
+    private static final String ATTENDANCE_CHECK_MENU = "1. 출석 확인";
+    private static final String ATTENDANCE_MODIFY_MENU = "2. 출석 수정";
+    private static final String RECORD_PRINT_FOR_EACH_CREW_MENU = "3. 크루별 출석 기록 확인";
+    private static final String DISMISSAL_CREW_CHECK_MENU = "4. 제적 위험자 확인";
+    private static final String QUICK = "Q";
+    private static final String PROMPT_TIME_INPUT_TO_MODIFY = "언제로 변경하겠습니까?";
+    private static final String PROMPT_DAY_INPUT_TO_MODIFY = "수정하려는 날짜(일)를 입력해 주세요.";
+    private static final String PROMPT_STUDENT_NAME_INPUT_TO_MODIFY = "출석을 수정하려는 크루의 닉네임을 입력해 주세요.";
+    private static final String PROMPT_STUDENT_NAME_INPUT = "닉네임을 입력해 주세요.";
+    private static final String PROMPT_START_TIME_INPUT = "등교 시간을 입력해 주세요.";
+    private static final String MENU_OPTION = "[1-4]|Q";
+    private static final String PRINT_TODAY_FORMAT = "오늘은 %d월 %d일 %s입니다. 기능을 선택해 주세요.\n";
+    private static final DateTimeFormatter dateTimeFormatterForHourMin = DateTimeFormatter.ofPattern("HH:mm");
+    private static final LocalTime START_TIME = LocalTime.of(8,0);
+    private static final LocalTime END_TIME = LocalTime.of(23,0);
+    private static final int DECEMBER_START_DATE = 1;
+    private static final int DECEMBER_ENT_DATE = 31;
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void setScanner(Scanner newScanner) {
+        scanner = newScanner;
+    }
+
+    private static void printMenu() {
+        System.out.println(ATTENDANCE_CHECK_MENU);
+        System.out.println(ATTENDANCE_MODIFY_MENU);
+        System.out.println(RECORD_PRINT_FOR_EACH_CREW_MENU);
+        System.out.println(DISMISSAL_CREW_CHECK_MENU);
+        System.out.println(QUICK);
+    }
+
+    public static String userInput(){
+        return scanner.nextLine();
+    }
+
+    public static String getUserWantMenu(TodayDate todayDate){
+        System.out.printf(PRINT_TODAY_FORMAT,todayDate.getTodayDate().getMonth().getValue(),todayDate.getTodayDate().getDayOfMonth(),todayDate.getTodayDay());
```

Comment:
get으로 계속 요청하지 않고 todayDate 객체에게 필요한 값을 바로 요청해보는게 좋을 것 같아요!
get을 계속 요청한다는 것은 객체가 책임을 다하지 않고 있다는 단서가 될 수 있습니다~

```suggestion
    System.out.printf(PRINT_TODAY_FORMAT, todayDate.getMonth(), todayDate.getDayOfMonth(), todayDate.getDayOfWeek());
```

Reply:
출력에 필요한 문자열 생성하는 것을 부탁하는 건 model 이 해야할 일이 아닐 것 같아서 현재 코드로 작성하였습니다.

찰리는 어떻게 생각하시는지 궁금합니다!!

Reply:
저는 문자열을 생성하는 것을 부탁하는것 보다는 내부 값을 편하게 꺼내올 수 있게 해준다 정도로 생각했어요~
getter 를 계속 호출하는것이 더 어색하다고 생각해서요 🤔 
`오늘 날짜 객체에게` 오늘 무슨 요일이야? 물어보면 되는 상황인데
오늘 날짜야 날짜좀 알려줘 -> 날짜야 오늘이 무슨달이야? 이렇게 물어보는것이 어색한 것 같아요 🤔 

말론이 말하는 출력을 위해 model 에 메서드를 하나 더 만드는 느낌도 동의해요!
저는 편의성 측면에서 허용하는 범위라고 생각했어요!
언젠가 사용할 수 있는 메서드이기도 하구요

하지만 말론이 생각하기에 view 를 위한 메서드라는것이 위험하다고 본다면
반영하지 않으셔도 됩니다!
저도 동의하는 부분이라서요 ㅎㅎ